Dockerfile

```diff
@@ -1 +1,77 @@
+# Use the official Ubuntu as the base image with ARG for version tag
+ARG UBUNTU_VERSION=latest
+FROM ubuntu:${UBUNTU_VERSION}
```

Comment:
Line 2 change the version to UBUNTU_VERSION=20.04 
Please remove the curly bracket on line 3